src/main/webapp/user-page.html

```diff
@@ -31,6 +31,15 @@
     </nav>
     <h1 id="page-title">User Page</h1>
 
+    <div id="about-me-container">Loading...</div>
```

Comment:
Any reason why you're moving the "about me" container up? It's best to leave unrelated changes in their own pull requests. You can reset by using the same "git checkout" command above (make sure to use the correct file name).